assignments/arrays.js

```diff
@@ -62,33 +66,79 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // }
 
 // ==== Challenge 1 ====
-// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// The dealer can't recall the information for a car with an id of 33 on his lot. 
+//Help the dealer find out which car has an id of 33 by logging 
+//the car's year, make, and model in the console log provided to you below:
+
+
+console.log("Car 33 is a " + inventory[32].car_model + " made by " + inventory[32].car_make + " and the year is " + inventory[32].car_year);
```

Comment:
This is hard coding which car and position in the array you are pointing at and it is a no-no. What if the position of that care shifted because another car has been sold?

What you want to do is iterate through each item in the array testing the `id` key on each object as you do, IF the `id` matches then you want to return that data.